### PR TITLE
3994 – Improve seeds file: add variation to requests

### PR DESCRIPTION
**Pull request by vasconsaurus**
## Description

Add some variation to tipline requests as per Brian's (request)[https://github.com/meedan/check-api/pull/1723#issuecomment-1802465942]

References: 3994

## How has this been tested?

Ran the seeds script to create an user, and to add data to the same user.

### CODE REVIEW

**Comment by vasconsaurus** [comment]
@brianfleming another one for you to take a look. I think it's the last one (for now) 🎉

**Comment by brianfleming** [comment]
So good @vasconsaurus I ran it, and now I see so much great stuff!

power

File: db/seeds.rb [hunk 1]
```diff
@@ -125,9 +111,22 @@ def create_project_medias_with_channel(user, project, team, data)
 end
 
 def create_tipline_user_and_data(project_media, team)
+  tipline_message_data = {
+    link: 'https://www.nytimes.com/interactive/2023/09/28/world/europe/russia-ukraine-war-map-front-line.html',
+    audio: 'https://media.smooch.io/apps/5c193e2380a9c90022e40b86/conversations/d445678ac87d2e9f44485e40/j5x5g1L28Y7fDrzU9DSwuwUR/wnHkwjykxOqU3SMWpEpuVzSa.oga',
+    video: 'https://media.smooch.io/apps/5c193e2380a9c90022e40b86/conversations/d445678ac87d2e9f44485e40/HEFL3cuzvNnCDXZd0ElYziN2/AOVFpYOfMm_ssRUizUQhJHDD.mp4',
+    image: 'https://media.smooch.io/apps/5c193e2380a9c90022e40b86/conversations/d445678ac87d2e9f44485e40/sUn5Sq9xRiacbV0Af2qmjl5G/bOoeoeV9zNA51ecial0eWDG6.jpeg',
```
Comment (caiosba) [inline db/seeds.rb:118]: Please replace these by fake URLs (we're moving out of Smooch anyway). The important bit here is just the file extensions, but the URLs don't need to be real.

File: db/seeds.rb [hunk 2]
```diff
@@ -170,14 +169,14 @@ def create_tipline_user_and_data(project_media, team)
     smooch_user_app_id: random_string,
     smooch_user_data: smooch_user_data.to_json
   }
-
+  
   Dynamic.create!(annotation_type: 'smooch_user', annotated: team, annotator: BotUser.smooch_user, set_fields: fields.to_json)
-
+
```
Comment (caiosba) [inline db/seeds.rb:174]: Trailing white space :)

**Review by caiosba — CHANGES_REQUESTED** [review]
Great stuff Manu, I just added two comments. One more note: we're using `sample` in a few places, which means that we won't necessarily get at least of example of each type (status, request links, request channel, etc.). I think it's fine for now, just noting that this is something we may change in the future if we really need to have at least one example of each type.

**Review by caiosba — APPROVED** [review]

**Pull request merged**